### ISSUE TITLE
Add a way of changing and converting EOL characters between different standards independent of OS

### ISSUE THREAD

**Issue by zaroth** [spyder-ide/spyder#2196]
As in title, I find myself moving files between different operating systems quite frequently (or even between cygwin-noncygwin environment in windows). I try to enforce unix EOL standard across all my projects, IDEs and OSes, and spyder doesn't let me do that.

Besides changing and converting EOL style of a single file (a nice implementation of this could be seen in Notepad++), a per-project setting and conversion would be useful.

**Comment by ccordoba12** [comment]
That's right. Spyder enfornces EOLs according to the operating system under which is running.

**Comment by desertdev** [comment]
+1 
Would be awesome to go into Preferences -> Editor -> Advanced and set line endings to Unix globally.

**Comment by ccordoba12** [comment]
Good idea. That will help with git merges and solve simple conflicts due to line endings.

@andfoy, please work on this one.
